docs/reference/commandline/dockerd.md

```diff
@@ -847,6 +847,17 @@ set like this:
     export DOCKER_TMPDIR=/mnt/disk2/tmp
     /usr/local/bin/dockerd -D -g /var/lib/docker -H unix:// > /var/lib/docker-machine/docker.log 2>&1
 
+Docker clients built with golang < 1.6 cannot talk to Docker daemons built
```

Comment:
I would add this to https://github.com/docker/docker/blob/master/docs/breaking_changes.md.